components/ws-manager/pkg/manager/create.go

```diff
@@ -336,6 +371,26 @@ func (m *Manager) createDefiniteWorkspacePod(startContext *startWorkspaceContext
 		},
 	}
 
+	for i, c := range pod.Spec.Containers {
+		if c.Name != "workspace" {
+			continue
+		}
+
+		pod.Spec.Containers[i].VolumeMounts = append(c.VolumeMounts,
+			corev1.VolumeMount{
+				MountPath: "/dev/net/tun",
+				Name:      "dev-net-tun",
+			},
+			corev1.VolumeMount{
+				MountPath:        "/.workspace",
+				Name:             daemonVolumeName,
+				MountPropagation: &mountPropagation,
+			},
+		)
+		pod.Spec.Containers[i].Command = []string{"/.supervisor/workspacekit", "ring0"}
+		break
+	}
```

Comment:
This should be folded into `createWorkspaceContainer`

Reply:
fixed